atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraLogHelper.java

```diff
@@ -62,13 +60,6 @@ static List<String> tokenRangesToServer(Multimap<Set<TokenRange>, CassandraServe
                 .collect(Collectors.toList());
     }
 
-    static Map<String, NonSoftFailureHostIdResult> idSupportingHostIdResultMap(
-            Map<CassandraServer, NonSoftFailureHostIdResult> cassandraServerMap) {
-        return EntryStream.of(cassandraServerMap)
-                .mapKeys(CassandraServer::cassandraHostName)
-                .toMap();
-    }
-
```

Comment:
This function doesn't check that if there are duplicated Cassandra host names in the cassandraServerMap. It can be the case that there are two different CassandraServer objects sharing the same name, which will cause the IllegalStateException

Reply:
We can instead return multimap here but then how do we differentiate between separate objects?